<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NAWI Graz Player</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0" colspan="3">
       <widget class="QSlider" name="slider1">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="volume1">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lautstärke:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3" colspan="3">
       <widget class="QSlider" name="slider2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="QVideoWidget" name="video1" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Lautstärke:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QVideoWidget" name="video2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Video 1 wählen</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Video 2 wählen</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="6">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="2">
         <widget class="QLCDNumber" name="total_time1"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="button_play_pause">
          <property name="text">
           <string>Play</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLCDNumber" name="total_time2"/>
        </item>
        <item row="0" column="3">
         <widget class="QLCDNumber" name="current_time2"/>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="current_time1"/>
        </item>
       </layout>
      </item>
      <item row="2" column="5">
       <widget class="QSlider" name="volume2">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Da&amp;tei</string>
    </property>
    <addaction name="actionChoose_Video_1"/>
    <addaction name="action_ffne_Video_2"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>I&amp;nfo</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionChoose_Video_1">
   <property name="text">
    <string>&amp;Öffne Video 1</string>
   </property>
  </action>
  <action name="action_ffne_Video_2">
   <property name="text">
    <string>Öffne &amp;Video 2</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">qvideowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
